src/snowcli/cli/snowpark/jobs/commands.py

```diff
@@ -70,7 +55,7 @@ def desc(id: str = typer.Argument(..., help="Job id"), **options) -> CommandResu
 def logs(
     id: str = typer.Argument(..., help="Job id"),
     container_name: str = typer.Option(
-        ..., "--container-name", "-c", help="Name of the container."
+        ..., "--container_name", "-n", help="Name of the container."
```

Comment:
Same here. We also want to reduce the amount of short flags as those are causing a lot of name clashes. You can consider making this argument not flag if this value is neccessary